### PR TITLE
[FIX] migrate_translations_to_jsonb: avoid ir_translation terms cleanup

### PR DESCRIPTION
**Pull request by chienandalu**
We're making use of core's `_get_translation_upgrade_queries` which aside from provide the proper queries to upgrade the translated fields to the new json structure tries to cleanup the former ir_tanslation records.

In OpenUpgrade we try to keep historic info after migration so we're able to quickly react to possible inconsistencies.

cc @Tecnativa TT49615

### CODE REVIEW

File: openupgradelib/openupgrade_160.py [hunk 1]
```diff
@@ -54,9 +53,14 @@ def migrate_translations_to_jsonb(env, fields_spec):
...
-                _get_translation_upgrade_queries(env.cr, field)
-            ):
+            # Ignore cleanup queries as we want to keep the original ir_translation
+            # table records in order to be able to fix possible inconsistencies once
+            # we're migrated.
+            migrate_queries, _cleanup_queries = _get_translation_upgrade_queries(
+                env.cr, field
+            )
+            for query in migrate_queries:
+
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_160.py:63]: Remove this empty line

**Comment by chienandalu** [comment]
There's a limitation here as the core method executes a query inside the method that expects the table `ir_translation` to be renamed as `_ir_translation`... so we we should do it in `base` `pre-migration` to be able to use it: https://github.com/OCA/OCB/blob/1e74b85c260fc0c6d15e7e4e13d9afcb9c2e9909/odoo/tools/translate.py#L1616-L1631

**Comment by pedrobaeza** [comment]
I recommend then that this method renames the table before, and restores it after.

**Comment by pedrobaeza** [comment]
Now that I see the diff, why previous approach replacing inside the query didn't serve?

**Comment by pedrobaeza** [comment]
OK, understood

@marielejeune FYI

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**